Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatUnreadMessagesCountDecorationView.swift

```diff
@@ -5,8 +5,8 @@
 import StreamChat
 import UIKit
 
-/// The date separator view that groups messages from the same day.
-open class ChatThreadRepliesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
+/// The view used to show a separator when there are unread messages.
+open class ChatUnreadMessagesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
```

Comment:
The 2 views are showing semantically different content. The new one shows unread count while the existing one shows number of replies. 

If we are to reuse this view for both cases, the name should something more generic to fit both cases like `ChatMessagesCountDecorationView`